pkg/node/manager.go

```diff
@@ -72,7 +72,8 @@ func UpdateNode(ni Identity, n *Node) {
 
 	nodeIP := n.GetNodeIP(false)
 	updateNodeCIDR(n.IPv4AllocCIDR, nodeIP)
-	updateNodeCIDR(n.IPv6AllocCIDR, nodeIP)
+	nodeIP6 := n.GetNodeIP(true)
+	updateNodeCIDR(n.IPv6AllocCIDR, nodeIP6)
```

Comment:
Ah I thought it was a bug. What happens if the node has a reachable IPv6? We keep assuming the node has a /112 (/96?) and all containers are directly reachable?